lib/telnyx/util.rb

```diff
@@ -39,17 +39,17 @@ def self.objects_to_ids(h)
     def self.object_classes
       @object_classes ||= {
         # business objects
+        AlphanumericSenderId::OBJECT_NAME       => AlphanumericSenderId,
+        AvailablePhoneNumber::OBJECT_NAME       => AvailablePhoneNumber,
+        Call::OBJECT_NAME                       => Call,
+        Conferences::OBJECT_NAME                => Conferences,
         Message::OBJECT_NAME                    => Message,
-        MessagingProfile::OBJECT_NAME           => MessagingProfile,
         MessagingPhoneNumber::OBJECT_NAME       => MessagingPhoneNumber,
-        MessagingSenderId::OBJECT_NAME          => MessagingSenderId,
-        MessagingShortCode::OBJECT_NAME         => MessagingShortCode,
```

Comment:
Is this the only place that it's using this class? I've noticed we commented all the tests too, so if that's the case we can erase ✂️ both the class and the test files.